Demo/Demo/SwiftUIComponents/PayPalVaultViews/PayPalVaultView.swift

```diff
@@ -15,13 +15,18 @@ struct PayPalVaultView: View {
                     )
                     SetupTokenResultView(vaultViewModel: paypalVaultViewModel)
                     if let setupTokenID = paypalVaultViewModel.state.setupToken?.id {
-                        Button("Vault PayPal") {
-                            Task {
-                                await paypalVaultViewModel.vault(setupTokenID: setupTokenID)
+                        ZStack {
+                            Button("Vault PayPal") {
+                                Task {
```

Comment:
random q: would it be cleaner if we created the `Task` within the `paypalVaultViewModel.vault()` method?

Reply:
Button actions are synchronous contexts, so I need to create a Task block there to call an async/await function.

Reply:
It makes sense what you are saying, make the views look cleaner. Let me add that to demo app refactor ticket.